Lib/pathlib/_abc.py

```diff
@@ -206,7 +206,11 @@ class PurePathBase:
     )
     pathmod = os.path
 
-    def __init__(self, *paths):
+    def __init__(self, *args):
+        paths = []
```

Comment:
Can you use a list comprehension here? E.g `paths = [path for path in args if path]`. Avoiding the call to append might be faster.

Reply:
Yes you're right.

I wrote it this way because I'm anticipating adding an `isinstance(arg, str)` check, but its premature.